processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -435,487 +419,58 @@ public Cursor apply(final Interval inputInterval)
                                         );
 
 
-                  final Offset initOffset = offset.clone();
+                  final Offset cursorOffset = offset.clone();
```

Comment:
can this be changed back to `initOffset` (or something more descriptive)? it is really hard to keep track of if `this.cursorOffset` or `cursorOffset` is the correct usage in this method.